frontend/src/pages/Admin/Glossary/index.tsx

```diff
@@ -11,30 +11,99 @@ export const Glossary: FC = () => {
     updateTitle("Glossary");
   }, [updateTitle]);
 
-  const { axiosAll } = useAxios();
+  /*
+   * The renderer is disabled for now to avoid dangerously setting HTML directly.
+   * instead, we are simply rendering it directly inside a div as a static component.
+   *
+   * If you would like to make the glossary editable, take the following steps:
+   *
+   * 1) uncomment the code below
+   * 2) find the HTML document at backend/docs/Glossary/Glossary.html
+   * 3) remove the static HTML inside the return() and uncomment the Renderer JSX line.
+   * 4) refresh and start editing the HTML file
+   *
+   *
+   */
 
   /**
    * Gets the glossary definition HTML.
    *
    * @returns {null|object}
    */
-  const getGlossary = async () => {
-    const response = await axiosAll().get(`/glossary`);
-    return { __html: response.data as string };
-  };
-
-  // Queries
-  const { data, isLoading } = useQuery([`glossary`], getGlossary, {
-    refetchOnWindowFocus: false,
-    retryOnMount: false,
-    refetchOnReconnect: false,
-    retry: false,
-    staleTime: Infinity,
-  });
+  // const { axiosAll } = useAxios();
```

Comment:
another way of maybe doing this is removing it all completely and adding a comment to indicate what commit it was changed
Not sure if this is problematic for the linter.

Reply:
Yes, that would work too, though I'd like to think it is kinder to the future maintainer to give them instructions and some lines of code to un-comment. 

I'm not entirely sure which approach is best, however, because I've yet to encounter a disabled feature I had to re-enable. Typically they just disappear forever.